### DISCUSSION TITLE
Visibility of first hit object is not fully raised when using Hidden and Target together

### DISCUSSION THREAD

**Discussion by ItsJustSomeDude** [ppy/osu#16216] [Q&A] [answered]
Check in settings, make sure the setting "Increase Visibility of first hit object when visibility impairment mods are enabled".

Start a map with just Hidden.  The first hit circle appears all at once, along with the first approach circle.  The first hit circle now fades away.  The following hit circles appear with no approach circles.  This is all right.

Start a map with just Target Practice.  The first hit circle appears all at once, with it's approach circle.  The following notes grow in from the background, with no approach circles.  This is all right.

Start a map with both Hidden and Target Practice.  The first hit circle appears all at once with no approach circle.  It then fades away.  This is not right, it should have still had an approach circle on the first circle.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/16258
